README.md

```diff
@@ -123,7 +123,13 @@ the ix modeling platform for scientific publications or technical reports:
 
 0. Make sure the R version installed is either 32 OR 64 bit (and >= 3.3.0), consistently with GAMS and Java. Having both 32 and 64 bit generates error.
 
-0. Install packages `rJava` , `devtools` and `optparse` via the R package manager
+0. Install packages, `devtools` and `optparse` via the R package manager
+
+0. Two alternative options are available:
+
+    - use the package developed in Python, requires the `reticulate` R package (it will allow to install/use `retixmp`)
```

Comment:
Thanks for testing it, I have not problems when solving in the tutorial, maybe we can check it together.
Yes I can write somewhere the difference in syntax between the two packages.

the dependences on other packages are only written in the readme, not installed automatically, because installing automatically all the R packages might create more problems. 
It was the same for rixmp and rmessageix and their dependences, if something is missing the installation is skipped. But we can think a better way to let the user know when the installation is not completed.